app/code/Mykhailok/AskAboutThisProduct/etc/config.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <trans_email>
+            <product_manager_support>
+                <recipient_email>product-support@example.com</recipient_email>
+                <recipient_name>Product Manager Support</recipient_name>
+            </product_manager_support>
+        </trans_email>
+        <catalog>
+            <review>
```

Comment:
@mykhailok-dv This is new functionality, and this config group is not suitable for such config. New group must be created.